primitives/core/src/assertion.rs

```diff
@@ -225,6 +225,9 @@ pub enum Assertion {
 	#[codec(index = 14)]
 	GenericDiscordRole(GenericDiscordRoleType),  // (generic_discord_role_type)
 
+	#[codec(index = 15)]
+	WeirdoGhostGangHolder,
+
```

Comment:
Maybe we should append the new VC to the end of the list, otherwise VCs previously generated with Index 15 would be decoded into this one and you'll see weird errors or even silently succeed but that's undesired - I think it's better to let it fail explicitly (like can't decode at all)

Reply:
Yes. We should only append from tail.